frame/grandpa/src/lib.rs

```diff
@@ -355,15 +365,25 @@ pub trait WeightInfo {
 	fn note_stalled() -> Weight;
 }
 
+/// Bounded version of `AuthorityList`, `Limit` being the bound
+pub type BoundedAuthorityList<Limit> = WeakBoundedVec<(AuthorityId, AuthorityWeight), Limit>;
+
 /// A stored pending change.
-#[derive(Encode, Decode, TypeInfo)]
-pub struct StoredPendingChange<N> {
+/// `Limit` is the bound for `next_authorities`
+#[derive(Encode, Decode, TypeInfo, MaxEncodedLen)]
+#[codec(mel_bound(Limit: Get<u32>))]
```

Comment:
sidenote: I am surprised it is needed, I though skipping the params didn't remove the normal bounds below.